### ISSUE TITLE
Make AlignParameters cop understand `def_delegators`

### ISSUE THREAD

**Issue by yous** [rubocop/rubocop#1247]
``` ruby
def_delegators :grade, :scores, :label, :color, :bg_color,
                       :to_s, :to_sym
```

RuboCop gives `C: Align the parameters of a method call if they span more than one line.`.

But making above code to

``` ruby
def_delegators :grade, :scores, :label, :color, :bg_color,
               :to_s, :to_sym
```

doesn't look good to me because it actually means:

``` ruby
def_delegators :grade, [:scores, :label, :color, :bg_color,
                        :to_s, :to_sym]
```
